.github/workflows/kube-linter.yaml

```diff
@@ -30,6 +30,7 @@ jobs:
             if [[ $(git status --porcelain) ]]; then
                 echo "Did you forget to build the manifests locally?" >&2
                 echo "Please run build-manifests.sh and update your PR" >&2
+                git diff >&2
```

Comment:
actually does this cause new files not added to show up in output?

Reply:
I'm not sure I am understanding what you mean

Reply:
does this cover the cause where a file was not checked into git ... as is the common cause of this issue.

Reply:
So like an entire file is missing from the `auto-generated` dir?

Reply:
yes. there be some `git diff` options needed. but we can improve overtime.

Reply:
You're right, I tested it on my other PR and if a file is missing, this extra line doesn't print anything out, so you are left with not knowing like before